dashboard/test/controllers/peer_reviews_controller_test.rb

```diff
@@ -73,7 +74,7 @@ class PeerReviewsControllerTest < ActionController::TestCase
     @peer_review.update(reviewer_id: plc_reviewer.id)
     post :update, params: {
       id: @peer_review.id,
-      peer_review: {status: 'accepted', data: 'This is great'}
+      peer_review: {status: SharedConstants::LEVEL_STATUS.accepted, data: 'This is great'}
```

Comment:
If you want, you can `include SharedConstants` in the test class, and then reference `LEVEL_STATUS` directly. Same below